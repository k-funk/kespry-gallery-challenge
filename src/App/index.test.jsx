import React from 'react';
import { render } from '@testing-library/react';
import Index from './index';


test('renders hello world', () => {
  const { getByText } = render(<Index />);
  const text = getByText(/hello world/i);
  expect(text).toBeInTheDocument();
});
