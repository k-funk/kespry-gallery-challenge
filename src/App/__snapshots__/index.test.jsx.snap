// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`outputs the expected tree when (default) 1`] = `
<div
  className="app"
>
  <h1>
    Image Gallery
  </h1>
  <div
    className="flex-grid"
  >
    <div
      className="col"
    >
      <AddImages
        addImage={[Function]}
      />
      <button
        className="btn-link"
        onClick={[Function]}
        type="button"
      >
        Fill Gallery
      </button>
    </div>
    <div
      className="col"
    >
      <Gallery
        imageList={
          Array [
            "https://picsum.photos/seed/1/450/300",
            "https://picsum.photos/seed/2/450/300",
            "https://picsum.photos/seed/3/450/300",
            "https://picsum.photos/seed/4/450/300",
            "https://picsum.photos/seed/5/450/300",
            "https://picsum.photos/seed/6/450/300",
            "https://picsum.photos/seed/7/450/300",
            "https://picsum.photos/seed/8/450/300",
            "https://picsum.photos/seed/9/450/300",
            "https://picsum.photos/seed/10/450/300",
            "https://picsum.photos/seed/11/450/300",
            "https://picsum.photos/seed/12/450/300",
            "https://picsum.photos/seed/13/450/300",
            "https://picsum.photos/seed/14/450/300",
            "https://picsum.photos/seed/15/450/300",
            "https://picsum.photos/seed/16/450/300",
            "https://picsum.photos/seed/17/450/300",
            "https://picsum.photos/seed/18/450/300",
            "https://picsum.photos/seed/19/450/300",
            "https://picsum.photos/seed/20/450/300",
            "https://picsum.photos/seed/21/450/300",
            "https://picsum.photos/seed/22/450/300",
            "https://picsum.photos/seed/23/450/300",
            "https://picsum.photos/seed/24/450/300",
            "https://picsum.photos/seed/25/450/300",
            "https://picsum.photos/seed/26/450/300",
            "https://picsum.photos/seed/27/450/300",
          ]
        }
      />
      <button
        className="btn-link"
        onClick={[Function]}
        type="button"
      >
        Empty Gallery
      </button>
    </div>
  </div>
</div>
`;
